libraries/eosiolib/contracts/eosio/privileged.hpp

```diff
@@ -185,6 +212,217 @@ namespace eosio {
     */
    void get_blockchain_parameters(eosio::blockchain_parameters& params);
 
+   /**
+    *  Helper structure for host functions set_parameters_packed and get_parameters_packed
+    *  @ingroup privileged
+    */
+   struct params_object{
+      std::string packed;
```

Comment:
Why not use a 'vector' here?  The packing is a binary blob and not a guaranteed representable string?  Also because of strings small object optimization it throws some holes in RVO at times.

Reply:
I removed this code, as the get_parameters_packed and set_parameters packed host functions were implemented in a different way.